buildSrc/src/main/java/Dependencies.kt

```diff
@@ -90,6 +90,10 @@ object Dependencies {
     const val androidx_core_ktx = "androidx.core:core-ktx:${Versions.AndroidX.core}"
     const val androidx_fragment = "androidx.fragment:fragment:${Versions.AndroidX.fragment}"
     const val androidx_lifecycle_extensions = "androidx.lifecycle:lifecycle-extensions:${Versions.AndroidX.lifecycle}"
```

Comment:
Looks like this is deprecated, does this still need to be here?

Reply:
No, I'll remove it